doctrine/doctrine-bundle/1.6/config/packages/dev/doctrine.yaml

```diff
@@ -0,0 +1,3 @@
+doctrine:
+    dbal:
+        profiling_collect_backtrace: true
```

Comment:
This option only exists in DoctrineBundle 1.11+, so it should not be in this version of the recipe.

Reply:
absolutely right, thank you 👍